src/TodoListItem.module.css

```diff
@@ -0,0 +1,15 @@
+.Container {
+  background-color: palegreen;
+}
+
+.ListItem {
+  color: blue;
+}
+
+.MainHeader {
+  color: pink;
```

Comment:
It's a little difficult to read the header because the pink and palegreen don't contrast with each other.  It can be helpful to use hex or HSL colors too! Here's a good color picker tool: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Colors/Color_picker_tool

For example you could do: `color: #A4A63A;`

You can also use a theme generator like this https://nikitahl.github.io/css-base/ to play around with colors.